src/react-app-env.d.ts

```diff
@@ -1 +1,45 @@
 /// <reference types="react-scripts" />
+
+export interface PostsFromServer {
+  userId: number,
+  id: number,
+  title: string;
+  body: string;
+}
+
+export interface CommentsFromServer {
+  postId: number;
+  id: number;
+  name: string;
+  email: string;
+  body: string;
+}
+
+export interface UsersFromServer {
```

Comment:
It's type for one user
```suggestion
export interface UserFromServer {
```